tests/plugins/autocomplete/autocomplete.js

```diff
@@ -65,18 +59,12 @@
 			editor.editable().fire( 'keyup', new CKEDITOR.dom.event( {} ) );
 
 			assertViewOpened( ac, true );
+			assert.isTrue( ac.view.getItemById( 1 ).hasClass( 'cke_autocomplete_selected' ) );
 
 			editable.fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 38 } ) );
 
 			assertViewOpened( ac, true );
-			assert.isTrue( ac.view.getItemById( 2 ).hasClass( 'cke_autocomplete_selected' ) );
-
-			editable.fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 38 } ) );
-
-			assertViewOpened( ac, true );
-
-			assert.isFalse( ac.view.getItemById( 2 ).hasClass( 'cke_autocomplete_selected' ) );
-			assert.isTrue( ac.view.getItemById( 1 ).hasClass( 'cke_autocomplete_selected' ) );
+			assert.isTrue( ac.view.getItemById( 3 ).hasClass( 'cke_autocomplete_selected' ) );
```

Comment:
Could we check another subsequent key up press? Here the selection cycled to the end of list (which is important case to test), we're missing the "regular" go up case.